src/query/storages/fuse/src/operations/mutation/recluster_mutator.rs

```diff
@@ -196,4 +196,14 @@ impl ReclusterMutator {
 
         Ok(false)
     }
+
+    // block1: [1, 2], block2: [2, 3]. The depth of point '2' is 1.
+    fn check_point(start: &[usize], end: &[usize]) -> bool {
```

Comment:
I‘m confused here, why this occasion's point depth is 1. Why it's special?? This will cause the panic??